python/pyspark/ml/evaluation.py

```diff
@@ -85,6 +88,9 @@ def _evaluate(self, dataset):
         self._transfer_params_to_java()
         return self._java_obj.evaluate(dataset._jdf)
 
+    def isLargerBetter(self):
+        return self._java_obj.isLargerBetter()
```

Comment:
This works for current use since it's called after evaluate has been called, but it could fail if a user calls it since the params may not have been transferred to java yet.  Call ```self._transfer_params_to_java()``` first.